docs/reference/networking.md

```diff
@@ -0,0 +1,51 @@
+# 5G Network Architecture
+
+5G was built with network functions divided by services which is known as the 5G core Service-Based Architecture (SBA). The 5G core is a network of interconnected services, as illustrated in the figure below.
+
+```{image} ../images/5g_network_architecture.png
+:alt: 5G Network Architecture
+:height: 600
+:align: center
+```
+
+The 5G Core Network is composed of various network functions, each serving a unique purpose. These functions communicate internally and externally over well-defined standard interfaces.
+The User Plane Function is a critical component of the 5G core network architecture. It oversees the management of user data during the data transmission process. The UPF serves as a connection point between the RAN and the data network.
+UPF Interfaces/reference points with employed protocols:
+- Access (N3): Interface between the RAN (gNB) and the UPF
+- Core (N6): Interface between the Data Network (DN) and the UPF
+- K8s LoadBalancer (N4): Interface between the Session Management Function (SMF) and the UPF  
+
+
+A minimal 5G network setup is given in the figure below.
+
+```{image} ../images/5g_network_diagram.png
+:alt: 5G Network Diagram
+:height: 600
+:align: center
+```
+
+According to the sample 5G network setup, the ports given in the below tables need to be allowed in the firewalls to assure the connectivity between Control Plane, User Plane and Radio Network.
+
+Connectivity between Control Plane and User Plane:
+
+|             |      **SOURCE**       |    **DESTINATION**    | 
+|:-----------:|:---------------------:|:---------------------:|
+|     IP      |     10.201.0.200      |      10.201.0.11      | 
+|   MODULE    |          UPF          |          SMF          |
+|  PORT NAME  |         PFCP          |         PFCP          |
+| PORT NUMBER |         8805          |                       |
+|  PROTOCOL   |          UDP          |          UDP          |
+|  INTERFACE  | K8s LoadBalancer(N4)  | K8s LoadBalancer(N4)  |
```

Comment:
We likely will have more IP's and ports to include here and we don't want 1 table per entry. I suggest having 1 row per firewall entry and two subcolumns for Module and Port under each top level column (source and destination). If you're not sure here lookup "Firewall table" on Google. 

Also here you introduced IP's like 10.201.0.200. I don't think we want to hardcode those.

The same comment applies to the other table below.

Reply:
TBH, I was not planning to add any IP address here, but after your comment in [Telco](https://chat.charmhub.io/charmhub/pl/jyj5b368fir858pb1sqkrauf8a) IPs together with a sample network diagram.  I strongly agree that we should not add ip addresses here.

Reply:
Done